src/Sylius/Bundle/CoreBundle/Fixture/OptionsResolver/LazyOption.php

```diff
@@ -133,4 +134,9 @@ public static function findOneBy(RepositoryInterface $repository, string $field)
             return $repository->findOneBy([$field => $previousValue]);
         };
     }
+
+    private static function findLimitedAll(RepositoryInterface $repository, int $limit = 2000): array
```

Comment:
I know that it may be annoying, that I'm just randomly jumping into this PR (and some changes are done due to my previous review), but after some time I would say that giving a limit here doesn't make any sense. It is not possible to use it anyway (class is final)

Reply:
Its ok :)

As for this function - it used inside `LazyOption` class (all `findAll`s replaced by this function call). Please see full diff for context:
https://github.com/Sylius/Sylius/pull/10506/files

Reply:
I get it. What I try to say, that adding a `limit` parameter here makes no sense, as it is not used anywhere. This function is private and it is used 4 times, always without second parameter

Reply:
Ah, you mean I should just inject that code directly. Will do that soon

Reply:
You can just remove this `limit` option here or propagate it's usage up to the functions that use them like
```php
public static function all(RepositoryInterface $repository, int $limit = 2000): \Closure
```

Reply:
Removed (replaced by private constant)